content/docs/state-and-lifecycle.md

```diff
@@ -155,23 +155,23 @@ Note how we pass `props` to the base constructor:
   }
 ```
 
-Class components should always call the base constructor with `props`.
+Les composants à base de classe doivent toujours appeler le constructeur de base avec `props`.
```

Comment:
```suggestion
Les composants à base de classe devraient toujours appeler le constructeur de base avec `props`.
```

On va préserver le conditionnel. En fait ce n'est **pas** obligatoire, mais [ça évite des surprises](https://overreacted.io/fr/why-do-we-write-super-props/).

Et d'ailleurs, pour être blindés, on devrait en fait récupérer puis passer *tous* les arguments, car il y en a trois (props, contexte, réconcilieur), mais bon, ils n'en parlent pas et c'est très avancé, donc nous non plus 😉

Reply:
Noté merci 😄